src/wui/wui.c

```diff
@@ -54,24 +54,28 @@ static void update_wui_vars(void) {
     osMutexRelease(wui_thread_mutex_id);
 }
 
-static int process_wui_request(char *request) {
-
-    if (strncmp(request, "!cip ", 5) == 0) {
-        uint32_t ip;
-        if (sscanf(request + 5, "%lu", &ip)) {
-            eeprom_set_var(EEVAR_CONNECT_IP4, variant8_ui32(ip));
+static int process_wui_request(wui_cmd_t *request) {
+
+    if (request->lvl == HIGH_LVL_CMD){
```

Comment:
Does CONNECT need set command?

Reply:
I did not find any documentation on that topic. We only know, that CONNECT will need a SET feature to set all the customizable variables, detailes are matter of disscusion.. It's my design used on connect ip4, token and hostname.